code/utils.py

```diff
@@ -124,11 +115,8 @@ def get_csv_paths(folder: Path) -> List[Path]:
     Returns:
         A list of the paths to the CSV files in the folder.
     """
-    return sorted([
-        folder.joinpath(f.name)
-        for f in folder.iterdir() if ((folder.joinpath(f.name).is_file()) and (
-            "csv" in f.name) and (".DS_Store" not in f.name))
-    ],
+    return sorted([folder.joinpath(f.name) for f in folder.iterdir() if (
+                (folder.joinpath(f.name).is_file()) and ("csv" in f.name) and (".DS_Store" not in f.name) and (f.suffix.casefold() in IMAGE_EXTS))],
```

Comment:
This should not check image_exts, as targeted files are csv. recommend to revert changes